ui/src/layout/SubMenu.js

```diff
@@ -30,19 +37,37 @@ const SubMenu = ({
   icon,
   children,
   dense,
+  secondaryLink = '',
```

Comment:
Default values should be set in `SubMenu.defaultProps`

Reply:
Ohh. okay. Will change.